MekHQ/src/mekhq/campaign/mission/enums/AtBContractType.java

```diff
@@ -237,107 +237,107 @@ public int generateEventType(Campaign campaign) {
             case RECON_RAID:
             case EXTRACTION_RAID:
                 if (roll < 10) {
-                    return AtBContract.EVT_BONUSROLL;
+                    return AtBEventType.BONUSROLL;
                 } else if (roll < 14) {
-                    return AtBContract.EVT_SPECIAL_SCENARIO;
+                    return AtBEventType.SPECIAL_SCENARIO;
                 } else if (roll < 16) {
-                    return AtBContract.EVT_BETRAYAL;
+                    return AtBEventType.BETRAYAL;
                 } else if (roll < 17) {
-                    return AtBContract.EVT_TREACHERY;
+                    return AtBEventType.TREACHERY;
                 } else if (roll < 18) {
-                    return AtBContract.EVT_LOGISTICSFAILURE;
+                    return AtBEventType.LOGISTICSFAILURE;
                 } else if (roll < 19) {
-                    return AtBContract.EVT_REINFORCEMENTS;
+                    return AtBEventType.REINFORCEMENTS;
                 } else if (roll < 20) {
-                    return AtBContract.EVT_SPECIALEVENTS;
+                    return AtBEventType.SPECIALEVENTS;
                 } else {
-                    return AtBContract.EVT_BIGBATTLE;
+                    return AtBEventType.BIGBATTLE;
                 }
```

Comment:
These, and all following if/else chains, can be simplified using switch/case with fall through.

Start at 20 and work down.

It'll increase readability for most.